aten/src/ATen/test/vulkan_api_test.cpp

```diff
@@ -20,7 +20,7 @@ bool checkRtol(const at::Tensor& diff, const std::vector<at::Tensor>& inputs) {
   constexpr float tolerance = 1e-5;
 #endif
 
-  return diff.abs().max().item<float>() < (tolerance * maxValue);
+  return diff.abs().max().item<float>() <= (tolerance * maxValue);
```

Comment:
Just curious, is this change required? Don't have any strong opinions on whether it should be `<` or `<=` but just curious why you changed it.

Reply:
If both inputs are all 0s, then `maxValue` becomes 0.0. As as a result, `almostEqual()` returns false even though both input tensors are identical (all 0s). I think we just never had this situation in tests before. BTW, multiplying `tolerance` by the max value from the tensor sounds not right to me but need to revisit this later since the same code has been used in many areas.